atc/db/build_test.go

```diff
@@ -377,6 +377,112 @@ var _ = Describe("Build", func() {
 			Expect(build.Status()).To(Equal(db.BuildStatusSucceeded))
 		})
 
+		Context("rerunning a build", func() {
+			var (
+				pdBuild, pdBuild2, rrBuild db.Build
+				err                        error
+				latestCompletedBuildCol    = "latest_completed_build_id"
+				nextBuildCol               = "next_build_id"
+				transitionBuildCol         = "transition_build_id"
+			)
+
+			Context("when there is a pending build that is not a rerun", func() {
```

Comment:
This phrasing gives me some trouble - I _think_ that 'being a rerun' is more exceptional than not being a rerun. So I'd rather have a test like this just say `when there is a pending build`, and if it's relevant that the pending build is a rerun, we call it out in those cases.

Unless we really need a word for 'not a rerun`, in which case can we invent one? an 'original build' or something like that?